### DISCUSSION TITLE
Minimize Dialog

### DISCUSSION THREAD

**Discussion by carlo1010** [primefaces/community#1063] [PrimeNG] [answered]
The function I'm requesting is the minimization of dialogs as happens on primeface jsf

[https://www.primefaces.org/showcase/ui/overlay/dialog.xhtml?jfwid=85e3b](url)
when creating entrerprice applications I often need to display more panels,

with this function I can have multiple panels at hand without having to search for all the sections I need every time

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

It's a good feature idea, we'll wait for community upvotes to implement it.

**Reply by m4n50n** [reply]
This option would be amazing, as I have had to do it by hand several times!
